pkg/catalogv2/git/content_test.go

```diff
@@ -0,0 +1,68 @@
+package git
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRelativeIconPath(t *testing.T) {
+	const baseRancherChart = "../rancher-data/local-catalogs/v2/rancher-charts/4b40cac650031b74776e87c1a726b0484d0877c3ec137da0872547ff9b73a721"
+	const rancherPublicURL = "https://git.rancher.io/charts"
+	testCases := []struct {
+		// input
+		test      string
+		base      string
+		publicURL string
+		path      string
+		// output
+		expectedContainingResult string
+		expectedSuffixResult     string
+		expectedError            error
+	}{
+		{
+			test: "#1.0: OK-istio", base: baseRancherChart, publicURL: rancherPublicURL, path: "file://assets/logos/istio.svg",
```

Comment:
Can you also add some negative cases in unit tests ?

Reply:
removed this unit-tests.
To make some negative test cases would be hard, and not worth the effort.
The main functionality is already being tested in the other unit-test.